### PR TITLE
chore: update package.json

### PR DESCRIPTION
**Pull request by rokroskar**
~~Use the same npm package name as before to be able to publish.~~

Fix up `package.json` and `.travis.yml` to be able to publish a scoped, public package.

### CODE REVIEW

**Comment by rokroskar** [comment]
actually, I may have a better solution... we may not need this

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
   "types": "lib/index.d.ts",
   "repository": {
     "type": "git",
-    "url": "https://github.com/SwissDataScienceCenter/renku-jupyterlab-ts.git"
+    "url": "git+https://github.com/SwissDataScienceCenter/renku-jupyterlab-ts.git"
```
Comment (lorenzo-cavazzi) [inline package.json]: You should probably not include `git+` in the url according to npm's specifics https://docs.npmjs.com/files/package.json#repository

Comment (rokroskar) [reply package.json]: oh really? `npm init` actually added it...

Comment (lorenzo-cavazzi) [reply package.json]: Well, it probably belongs there after all. I believe the code more than the documentation :smile:

**Comment by rokroskar** [comment]
well it turns out pushing `@renku/renku-jupyterlab-ts` means for some reason that we need a paid account for private packages... so lets merge this and keep the name as it was before (at some point during the last few weeks it was changed to `@renku/renku-jupyterlab-ts`)

**Comment by rokroskar** [comment]
ah, or we need this: https://docs.npmjs.com/creating-and-publishing-scoped-public-packages#publishing-scoped-public-packages

File: .travis.yml [hunk 2]
```diff
@@ -18,31 +18,26 @@
...
-          tags: true
+  - stage: npm release
+    if: tag IS present
+    node_js: '8'
+    script: yarn build
+    deploy:
+      provider: npm
+      email: "$NPM_EMAIL"
+      api_key:
+        secure: LQVoQuUin+3CiHPbCUaHb6l38rLDemkqFF4PoVJZNym0zv1WPuGPErOxIEcID/ncONjdKgZh1ZmV8Y9YF/PbHvGwx18W0UKu3fHewMcJjd0rlGSFBsYHY33A2aAdP86cbmQ19mkdQSFrEB2vDhE6m4yYmoQCWtUg1LLzVg2qFTX625XQ3FFxtmDESMJd2z/9TxOcmrBbPw78sqq4XjYw4qTdxGNGfovzYZeMyTtMTZCzimFnpC/serlL5SCgW/ELMslO5jQKK7+TbpikvygwHfTRez+nmuOGL0ARioRTM9bP1/1UZtSBUHFUDtLFLw45PkDmYcoH5i04iI7peh2yl4nEFfmntpl8eVBGFHDCyfpD8SGpfP6Cv0CWjRULcOgrRz92aupMzPwbse4QkozV9iicNa2lUQ/WV8/AJWML24phRLNXm2fNcb2q+5pTNRHq7oUclinwdxJUw28161LwyF4YQsNd6ixNhMrxAZAWueyboOZalXkMV7mcIe8tY52JYjzib36JRkAVDBshbcZZJ3zgG+NbBeghKjtNTR0xiBGjEvbcM10J4WNzM/RTlgbZheV0KNOrdMabM9HwShBV+gSAVq6SF0AM/lKevQsKUGm39rmnlzOMEYVk9SUUOLdUg7fqJHI1YGrsPbeIZW4wGFeJ7bOe8rj9sb8AlUfW0ew=
```
Comment (lorenzo-cavazzi) [inline .travis.yml]: I assume this is encrypted and a deployer needs to somehow decrypt this to get the real oauth token to be able to publish, right?

**Pull request merged**